### ISSUE TITLE
Ensures headings have discernible text

### ISSUE THREAD

**Issue by andrmaz** [andrmaz/dash-journal#24]
Fix any of the following:\n  Element does not have text that is visible to screen readers\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  Element has no title attribute
